src/lib.rs

```diff
@@ -24,23 +24,16 @@ pub type Result<T> = core::result::Result<T, FfaError>;
 
 pub fn u64_to_uuid(high: u64, low: u64) -> Uuid {
     let mut bytes = [0u8; 16];
-    bytes[..4].copy_from_slice(&high.to_be_bytes()[4..]);
-    bytes[4..8].copy_from_slice(&high.to_be_bytes()[..4]);
-    bytes[8..12].copy_from_slice(&low.to_be_bytes()[4..]);
-    bytes[12..].copy_from_slice(&low.to_be_bytes()[..4]);
+    bytes[..8].copy_from_slice(&high.to_le_bytes());
+    bytes[8..].copy_from_slice(&low.to_le_bytes());
     Uuid::from_bytes(bytes)
```

Comment:
Actually, turns out there's a better version:

```suggestion
Uuid::from_u64_pair((high, low))
```

See https://docs.rs/uuid/latest/uuid/struct.Uuid.html#method.from_u64_pair

Reply:
I've removed this wrapper function totally and directly called as_u64_pair and from_u64_pair in msg\mod.rs